.github/scripts/verifyActions.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Re-compiles all Github Actions and verifies that there is no diff,
```

Comment:
NAB: This comment is a bit confusing because this script is not doing the compiling.

Reply:
Good point, updated!

Reply:
Actually I ended up moving the compiling into this script and leaving this comment as-is.